src/app/components/shared/datetime/datetime/datetime.component.ts

```diff
@@ -0,0 +1,132 @@
+import { Component, Input } from "@angular/core";
+import { isInstantiated } from "@helpers/isInstantiated/isInstantiated";
+import { TimezoneInformation } from "@interfaces/apiInterfaces";
+import { DateTime, IANAZone, Zone } from "luxon";
+import { AbstractDatetimeComponent } from "../abstract-datetime.component";
+
+type BawTimezoneUnion = Zone | TimezoneInformation | string;
+
+/**
+ * Displays a datetime (possibly from another timezone) in the users locale and timezone
+ * Commonly used for displaying client side timestamps e.g. Project created at: xyz
+ *
+ * This component does not assume that all dates input are utc, and will localize using ISO8601 offset information if possible
+ */
+@Component({
+  selector: "baw-datetime",
+  templateUrl: "../abstract-datetime.component.html",
+})
+export class DatetimeComponent extends AbstractDatetimeComponent {
+  public constructor() {
+    super();
+  }
+
+  @Input({ required: true }) public value!: DateTime | Date;
+  @Input() public dateOnly: string | boolean;
+  @Input() public timeOnly: string | boolean;
+  protected timezone?: BawTimezoneUnion;
+
+  // we create a getter for the a luxon DateTime object so that we can get values from the @Input() value
+  // without type guards, shape, and instance checks
+  protected get luxonDateTime(): DateTime {
+    const luxonDate: DateTime =
+      this.value instanceof Date
+        ? DateTime.fromJSDate(this.value, { zone: "utc" })
```

Comment:
`"utc"` is not the name of the Zone. Why is it lower cased? Can we use `FixedOffsetZone.utcInstance` instead?

Reply:
I'm also wondering why we have to specify zone at all - especially since we just `.toLocal` 4 lines down...

Reply:
By the looks of it, we don't need the { zone: "utc" }

> "utc" is not the name of the Zone

Funny enough, this luxon `{ zone }` [options object doesn't actually have to be a zone (or even an IANA identifier!)](https://github.com/moment/luxon/blob/3125686af82d9a25c7267a1cf1eb838a3d41144f/docs/zones.md?plain=1#L55-L58) (weird in my opinion)

This is also the reason why I've had to implement a helper function to check if the `zoneName` returned by a zone object is actually an IANA identifier, or if Luxon has used one of their custom identifiers

![image](https://github.com/QutEcoacoustics/workbench-client/assets/33742269/b56a77c1-c6d0-4324-8e9c-ae0fd9ce7eb7)

Reply:
I didn't get time to understand that isIanaTimezone function but it was a huge code smell. My gut instinct (without fully understanding everything here) is that we should just leave this to luxon.

Reply:
I was able to replace this function with `IANAZone.IsValidZone`